src/sage/combinat/posets/hasse_cython.pyx

```diff
@@ -71,7 +71,7 @@ class IncreasingChains(RecursivelyEnumeratedSet_forest):
             self._greater_than = positions
             self._vertices = list(range(n))
 
-        self._element_constructor = element_constructor
+        self._constructor_ = element_constructor
```

Comment:
This name `_constructor_` so far does not appear anywhere in our codebase. Why is this an improvement?

Reply:
because it is now a specific name used only in this context, not related to any general mechanism

Reply:
But why use `_constructor_`, not `_constructor`?

Reply:
better now ?